data/json/obsoletion_and_migration_0.I/migration_items.json

```diff
@@ -1221,5 +1221,15 @@
     "id": "riv_suppressor",
     "type": "MIGRATION",
     "replace": "suppressor"
+  },
+  {
+    "id": "surv_six_shooter",
+    "type": "MIGRATION",
+    "replace": "sw_619"
+  },
+  {
+    "id": "surv_hand_cannon",
+    "type": "MIGRATION",
+    "replace": "sw_619"
```

Comment:
Wouldn't it be a cleaner migration to move to stuff in the same caliber?

Reply:
> Wouldn't it be a cleaner migration to move to stuff in the same caliber?

What gun would be better? I don't imagine migrating to a 1911 would be any better, as it has no internal magazine